api/auth_middleware.py

```
@@ -0,0 +1,133 @@
+    """
+    if not _firebase_initialized:
+        raise HTTPException(
+            status_code=503,
+            detail="Firebase authentication is not configured"
+        )
+    
+    token = credentials.credentials
+    
+    try:
+        decoded_token = auth.verify_id_token(token)
+        return decoded_token
```

Ruff (0.14.3)

Consider moving this statement to an `else` block

(TRY300)